libobs/obs-data.c

```diff
@@ -1287,6 +1287,11 @@ double obs_data_get_double(obs_data_t *data, const char *name)
 	return obs_data_item_get_double(get_item(data, name));
 }
 
+double obs_data_get_float(obs_data_t *data, const char *name)
```

Comment:
Are you sure this is supposed to be `double [...]_get_float(`?

Reply:
not entirely sure, after all it will still be double precision if it's just a wrapper over _get_double